Makefile

```
@@ -9,7 +9,7 @@ AUTOINST :=
 NAME_PREFIX := nvmeof-
 
 # URL for downloading OVMF image
-OVMF_URL := https://github.com/timberland-sig/edk2/releases/download/release-3f693ec/timberland-ovmf-release-3f693ec.zip
+OVMF_URL := https://github.com/timberland-sig/edk2/releases/download/release-9e63dc0/timberland-ovmf-release-9e63dc0.zip
```

@Douglas-Farley which edk2 commit corresponds to this change?  I take it that this edk2 change introduced a new dependency on VConfig.efi .

---

It's from https://github.com/timberland-sig/edk2/pull/8. It is no new dependency. VConfig is part of EDK2, it's just not built by default. I just changed the setup such that it's now built and packaged, just like `NvmeOfCli.efi`.

---

IF you're willing to have it be dynamic and account for future patches, you \_could_ also collect this url the fancy way...

```suggestion
OVMF_URL := $(shell curl -s https://api.github.com/repos/timberland-sig/edk2/releases/latest | jq -r '.assets[] | select(.name|startswith("timberland-ovmf-release")).browser_download_url')"
```

This would have been easier if we didn't embed the hash in the file name... oops...

---

Thanks for pointing it out. I actually have a [footnote in the README](https://github.com/timberland-sig/suse-linux-poc#user-content-fn-ovmf_url-2a0b24459eb662123e82caf9e317c10e) showing how to do that.
But when I tested it, I experienced strange errors from GitHub saying something like my user's quota for API access were exceeded. Therefore I refrained from putting it into the automated scripts, to avoid that people might run into just that error.
I haven't seen these errors again, but it's a kind of problem I can't control, so I prefer to stick with the hard-coded release number for the time being.